org.opent2t.sample.multisensor.superpopular/com.wink.sensorpod/js/thingTranslator.js

```diff
@@ -84,7 +89,7 @@ function convertDeviceBatteryToTranslatorBattery(batteryValue) {
  * Returns a default value if the specified property is null, undefined, or an empty string
  */
 function defaultValueIfEmpty(property, defaultValue) {
-    if (property === undefined || property === null || property === "") {
+    if (!property) {
```

Comment:
This check (`if (!property)`) might use the default value in unexpected situations. If the property you're reading is a number, we would use the default if you're reading the value '0', or if the property you're reading is a Boolean value, we would use the default if you're reading the value 'false'.

https://developer.mozilla.org/en-US/docs/Glossary/Falsy